src/services/api.ts

```diff
@@ -62,6 +62,21 @@ export class Api {
         })
     }
 
+    async getUserDetails(userId) : Promise<any> {
```

Comment:
we can probably do beter typin than Promise<any> 
Something like Promise<User> maybe?

Reply:
Yes instead of `any` `unknown`. Ideally share types between `functions` and `front`